gserver/commands.go

```diff
@@ -73,6 +74,14 @@ func newSeedCommand() *cobra.Command {
 
 			fmt.Fprintf(cmd.ErrOrStderr(), "Seed running at multiaddrs: %s\n", joinedAddrs)
 
+			c := make(chan os.Signal, 1)
```

Comment:
Could you raise this section up to https://github.com/gordian-engine/gcosmos/blob/629f0ff48d7db6206f0c5aa1c00a2adad62bc4d6/main.go#L27-L28 so that all commands inherit interrupted contexts? And use [`signal.NotifyContext`](https://pkg.go.dev/os/signal#NotifyContext), which actually exists, unlike the imaginary type I put in the linked comment.

Reply:
`srvcmd.Execute` builds it's own `context.Background()`, so had to copy the `Execute` function into main.go and make it take a context.